k-distribution/include/kframework/ktest-fail.mak

```diff
@@ -24,7 +24,7 @@ kompile: $(TESTS)
 dummy:
 
 %.k %.md: dummy
-	$(KOMPILE) $(KOMPILE_FLAGS) --backend $(KOMPILE_BACKEND) $(DEBUG_FAIL) $@ -d $(DEFDIR) 2>&1 | sed 's!'`pwd`'/\(\./\)\{0,1\}!!g' $(CHECK) $@.out $(CHECK2)
+	$(KOMPILE) $(KOMPILE_FLAGS) --backend $(KOMPILE_BACKEND) $(DEBUG_FAIL) $@ -d $(DEFDIR) 2>&1 | sed 's!'`pwd`'/\(\./\)\{0,2\}!!g' $(CHECK) $@.out $(CHECK2)
```

Comment:
What's the explanation behind this one?

Reply:
On my machine, the paths from reported warnings include a `/././` and on Jenkins they only include `/./`.
I'm not sure why. It might be a java thing. This change takes into consideration both cases.